circuitbreakerpanel/circuitbreakerpanel-java-mvn/.gitignore

```diff
@@ -0,0 +1,17 @@
+target
```

Comment:
You can remove this `.gitignore` file, since there is already one in the top-level directory.

Reply:
Is it missing ones for files that are actually created? The default one has a lot of extra stuff in it.

Reply:
I think those should be added to the top-level `.gitignore`, then, rather than having project-specific overrides.

Reply:
+1 to adding them to https://github.com/lagom/lagom-recipes/blob/master/.gitignore